src/Microsoft.VisualStudio.Editors/Microsoft.VisualStudio.Editors.pkgdef

```diff
@@ -282,67 +281,67 @@
 "codeBase"="$PackageFolder$\Microsoft.VisualStudio.CoreUtility.dll"
```

Comment:
Apparently this pkgdef gets dropped in Common7\IDE\CommonExtensions\Microsoft\Editor, but not the dll, this is incorrect now that we pulled both of these out of VS. Will figure this out today why we have these dependent assemblies anyway - they were introduced in VS 15.